Packs/CortexXDR/Scripts/InvestigationVerdict/InvestigationVerdict.yml

```diff
@@ -0,0 +1,19 @@
+comment: This widget displays the Verdict.
+commonfields:
+  id: InvestigationVerdict
```

Comment:
need to change the name as my previous comment

Reply:
Done